pkg/gvk/gvk.go

```diff
@@ -180,3 +180,40 @@ func (x Gvk) IsClusterKind() bool {
 	}
 	return false
 }
+
+var notNamespaceableKinds = []string{
```

Comment:
use this or clusterLevelKinds, but not both.
this looks like a a much more complete list than we previously had, thanks!

Looks like `IsClusterKind()` is morally equivalent to `!IsNamespaceable()`
please remove one or the other. I don't have a preference.

Reply:
sorry that's the last nit.

Reply:
@monopole This done. Sorry this was my fault in a first place. When investigating the IsNamespeable implementation I found the IsClusterKind implementation and did not think to merge the two functions.